Components/Config.php

```diff
@@ -77,6 +82,39 @@ public function get($key = null, $default = null)
         return $this->data;
     }
 
+    /**
+     * @param array $config
+     * @param string $name
+     * @return array
+     */
+    private function addInheritedConfig(array $config, $name)
+    {
+        // check if inherited or null fields are in config to get from default shop
+        if (!in_array(self::INHERITED_CONFIG_VALUE, $config) && !in_array(null, $config)) {
+            return $config;
+        }
+
+        // get default shop
+        /** @var Repository $shopRepository */
+        $shopRepository = Shopware()->Models()->getRepository('Shopware\\Models\\Shop\\Shop');
```

Comment:
Shop::class instead of 'Shopware\\Models\\Shop\\Shop'?

Reply:
will do tomorrow